### ISSUE TITLE
socket test fails with SEGV on Ubuntu

### ISSUE THREAD

**Issue by ktakashi** [ktakashi/sagittarius-scheme#137]
**[Original report](https://bitbucket.org/ktakashi/sagittarius-scheme/issues/132) by Takashi Kato (Bitbucket: [ktakashi](https://bitbucket.org/ktakashi), GitHub: [ktakashi](https://github.com/ktakashi)).**

----------------------------------------

Socket test sometimes fails on Ubuntu (only happens on drone.io environment for now).

Following environments don't happen (for now):

- Ubuntu 14.04 (x64)
- Lubuntu 15.04 (x64, vbox)

Error log from drone.io
```
%%%% Starting test (run-socket-test)
make[3]: *** [CMakeFiles/test] Segmentation fault
make[2]: *** [CMakeFiles/test.dir/all] Error 2
make[1]: *** [CMakeFiles/test.dir/rule] Error 2
make: *** [test] Error 2
```

**Comment by ktakashi** [comment]
**Original comment by Takashi Kato (Bitbucket: [ktakashi](https://bitbucket.org/ktakashi), GitHub: [ktakashi](https://github.com/ktakashi)).**

----------------------------------------

NOTE: drone.io is Ubuntu 12.04.

**Comment by ktakashi** [comment]
**Original comment by Takashi Kato (Bitbucket: [ktakashi](https://bitbucket.org/ktakashi), GitHub: [ktakashi](https://github.com/ktakashi)).**

----------------------------------------

Should be fixed by <<cset: decb414>>
